scripts/linux/runX509Tests.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
```

Comment:
How is this run in E2E tests?

Reply:
It will be invoked as part of the E2E tests, which is not part of the iotedge repo.